src/hooks/eventsCustomHooks.tsx

```diff
@@ -54,3 +61,80 @@ export const useFetchEvents = (refresh: boolean = false) => {
 
   return { events, loading, error };
 };
+
+export const useEventSaves = (eventId: string) => {
```

Comment:
Seems like we have similar functionality, albeit difference implementation, for these user save handlers. If you want, it would be cleaner to have a middle man in between that can be used to call the appropriate endpoint depending on the type of object calling or parameters passed. Again definitely not needed but might be something you'd be interested in.

Reply:
I like this but any ideas how to organize it? rn we have event saves handled in eventAPI/hooks, marketplace saves handled in marketplaceAPI/hooks, etc.

 I could make a middle man function that calls the appropriate version based on the type but not sure how I would structure it, i.e. what file it would belong to